hardhat.config.ts

```diff
@@ -8,12 +8,12 @@ import "@nomiclabs/hardhat-ethers";
 const config: HardhatUserConfig = {
   solidity: '0.6.12',
   starknet: {
-    dockerizedVersion: "0.8.2", // alternatively choose one of the two venv options below
+    // dockerizedVersion: "0.8.2", // alternatively choose one of the two venv options below
```

Comment:
The changes done in this file aren't needed for the tests to work. You should revert them.